### PR TITLE
Added aliases and Reactotron

### PR DESCRIPTION
**Pull request by nathan-hyan**
## Summary

Added Reactotron and aliases

## Trello Card

https://trello.com/c/yxge2Zkj

### CODE REVIEW

File: wbooks/tsconfig.json [hunk 1]
```diff
@@ -57,7 +57,17 @@
...
+      "@constants/*": ["src/constants/*"],
+      "@assets/*": ["./src/app/assets"],
+      "@components/*": ["./src/app/components"],
+      "@hooks/*": ["./src/app/hooks"],
+      "@screens/*": ["./src/app/screens"],
+      "@config/*": ["./src/config"],
+      "@interfaces/*": ["./src/interfaces"],
+      "@redux/*": ["./src/redux"],
+      "@services/*": ["./src/services"],
+      "@utils/*": ["./src/utils"],
```
Comment (sfernandez11) [inline wbooks/tsconfig.json:69]: You forgot the `/*` at the end of the lines

**Review by sfernandez11 — COMMENTED** [review]
Just one comment

File: wbooks/tsconfig.json (hunk 1 above)
Comment (nathan-hyan) [reply wbooks/tsconfig.json]: Fixed

**Review by sfernandez11 — APPROVED** [review]
🥇

**Pull request merged**